Enable functionality to use common schema in model database

jodybuz opened the pull request:
A new variable has been added use_common_masking_policy_schema_only which allows for a common schema within the model database to be specified for storing masking policies.

(Similar to existing functionality which uses use_common_masking_policy_db to set a common database and schema to store masking policies.)

entechlog commented:
@jodybuz , Thank You for the PR. I will review/test and get back. 

- Can you also update the README.md with the details on how to use this new feature ?
- I also see changes to "integration_tests/dbt_project.yml", was that required ?
-

jodybuz commented:
Thanks @entechlog - of course, I will update the README with those details this morning.

Regarding the updates to the "integration_tests/dbt_project.yml" file - I added a commented out version of my new variable so it could be tested. But the other reformatting changes were added by my yaml linter during the commit. Happy to reverse those changes if thats helpful?

entechlog commented:
Thank You @jodybuz , I will test this and get it merged

jodybuz commented:
Hi @entechlog - Sorry, but I've added one last change into this PR (hope it's not too late.) We have custom materializations in our dbt project, so I've added a parameter which allows your package to apply masking policies to these types of models. I've updated the README with instructions on how to use this feature too.

entechlog commented:
@jodybuz , Sure. Sorry I have been busy for last couple of days. I will try to take a look at them today evening

nadesansiva commented:
@jodybuz , The tests ran fine, so going a head and merging this now.

jodybuz commented:
Thanks guys - this is going to be so useful for us (and hopefully other too.)
